### PR TITLE
Add user personas to the docs

### LINKED ISSUE #12: Add Personas to the documentation

### PR DESCRIPTION
**Pull request by baer**
Closes #12

### CODE REVIEW

File: docs/personas.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+**Andrew**: A non technical person who needs to see the overall health of the project. Andrew is currently only judged by his boss on shipping features, which are easily measurable based on completed stories in JIRA. This report exists partially to create quantitative metrics about quality for him to be judged on.
+
+**Barbara**: Barbara is a core web engineer who is savvy with both the report card and the tools it uses under the hood. She will use the information to both help justify their ongoing costs and to help plan where the engineering and advocacy efforts need to be focused. Barbara will be focused more on the reports cards efficacy and advocating its use than consuming the report directly since ideally she will not be in a position to call somebody out for having a bad report.
```
Comment (patrickkettner) [inline docs/personas.md]: I don't think `core web` translate well outside of WM

Comment (baer) [inline docs/personas.md]: True story, I didn't think about that. It could roughly translate to tech/project lead or engineering manager.

Comment (patrickkettner) [inline docs/personas.md]: I'd probably just call it a senior dev

Comment (baer) [inline docs/personas.md]: I'll change it to señor dev and we can update as we go

Comment (patrickkettner) [inline docs/personas.md]: plzdo

File: docs/personas.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+**Greg**: Greg is a developer on a track. He really wants to write high quality well tested code but is frequently under tight deadlines and pressure from management to get features out the door. He’d love to see his track use better practices but doesn’t feel like he has the tools to advocate for his cause. He wants a tool that can show him the health of particular areas of the code so that he can advocate for refactoring as well as justify the time it takes to write a feature.
```
Comment (baer) [inline docs/personas.md]: I'm also going to remove the reference to tracks.

**Comment by baer** [comment]
@patrickkettner: Have another peak - I think we're all set

**Comment by patrickkettner** [comment]
double plus good

:shipoopi:

**Pull request merged**